templates/worker/01-worker-kubelet/_base/units/kubelet.yaml

```diff
@@ -20,7 +20,7 @@ contents: |
         --container-runtime=remote \
         --container-runtime-endpoint=/var/run/crio/crio.sock \
         --allow-privileged \
-        --node-labels=node-role.kubernetes.io/worker \
+        --node-labels=node-role.kubernetes.io/worker,node.openshift.io/os_version=${OS_VERSION} \
```

Comment:
Why not just add `/etc/os-release` as an EnvironmentFile and pull it out from there?

Reply:
that does simplify things. fixed!